src/containers/pages/InterventionPlan/UpdatePlan/tests/fixtures.ts

```diff
@@ -83,6 +84,7 @@ export const updatePlanFormProps = {
     start: parseISO('2019-07-10T00:00:00.000Z'),
     status: 'active',
     taskGenerationStatus: 'False',
+    teamAssignmentStatus: '',
```

Comment:
What are the possible values of this?

Reply:
I don't think we need them because we are only updating based on the values that we get from the server that is why have used `string`.

Reply:
Understood.  But what are the possible values of this for plan objects in general?

Reply:
`False` and `True`